src/chains/algorand_1/templates/chainActions/chainSpecific/asset_config.ts

```diff
@@ -0,0 +1,53 @@
+import * as algosdk from 'algosdk'
+import {
+  AlgorandDecomposeReturn,
+  AlgorandChainActionType,
+  AlgorandSuggestedParams,
+  AlgorandTransactionTypeCode,
+  AlgorandTxAction,
+  AlgorandTxActionRaw,
+  AlgorandActionAssetConfigParams,
+} from '../../../models'
+import { AlgorandActionHelper } from '../../../algoAction'
+import { isNullOrEmpty } from '../../../../../helpers'
+
+/**
+ * Composes asset config action */
+export const composeAction = (args: AlgorandActionAssetConfigParams, suggestedParams: AlgorandSuggestedParams) => {
+  const argsEncodedForSdk = new AlgorandActionHelper(args as AlgorandTxAction).actionEncodedForSdk
+  const { from, note, assetIndex, assetManager, assetReserve, assetFreeze, assetClawback } = argsEncodedForSdk
+  const composedAction = algosdk.makeAssetConfigTxnWithSuggestedParams(
+    from,
+    note,
+    assetIndex,
+    assetManager,
+    assetReserve,
+    assetFreeze,
+    assetClawback,
+    suggestedParams,
+    args.strictEmptyAddressChecking,
+  )
+  return { ...composedAction }
+}
+
+export const decomposeAction = (action: AlgorandTxAction | AlgorandTxActionRaw): AlgorandDecomposeReturn => {
+  const actionHelper = new AlgorandActionHelper(action)
+  const actionParams = actionHelper.paramsOnly
+  // Identify chainActionType using type
```

Comment:
Since there is more than one action using AssetConfig type, I'd add a clear comment to each of them stating that. e.g. AssetConfig type is used for several chain actions - so checking for other params to determine which action